macros/get_messaging_service_columns.sql

```diff
@@ -1,6 +1,3 @@
---To disable this model, set the using_domain_names variable within your dbt_project.yml file to False.
-{{ config(enabled=var('using_twilio_messaging_service', True)) }}
-
```

Comment:
This macro is not called if the staging model is disabled. Therefore, we can remove this extra config.

Reply:
Gotcha, thanks for noticing